package.json

```diff
@@ -7,7 +7,7 @@
   "repository": "pandell/SlickGrid",
   "author": "Michael Leibman <michael.leibman@gmail.com>",
   "dependencies": {
-    "jquery": "*",
+    "jquery": "^2.2",
```

Comment:
I think we should keep this consistent with `web-pli` and pin it to `~2.1`. Especially due to pandell/web-pli#231